src/components/assets/drawer.css

```diff
@@ -205,3 +205,27 @@
 .ball.active {
   transform: translateX(20px);
 }
+
+
+@media only screen and ( max-width: 768px) {
+  .menu-drawer {
+    height: 100%;
+    width: 0;
+    position: fixed;
+    z-index: 1;
+    top: 10;
+    left: 0;
+    background-color: #1d1f22;
+    overflow-x: hidden;
+    transition: 0.2s;
+    display: flex;
+    flex-direction: column;
+    padding-top: 2rem;
+  }
+  .menu-drawer.active {
+    /* display: block; */
+    /* width: 80vw; */
+    width: 21vw;
```

Comment:
With this value  (/* width: 80vw; */)  the opening and closing menu looks amazing but however the closing menu cross icon does not appear when it is active in fact i can suggest to create one with JavaScript (or JSX as we are using react) instead of generating it by using Bootstrap.
 Please Just implement a method once the user clicks outside of the menu container and add a cross closing icon without bootstrap, this will work amazingly fine.

take a look at the layout below before the value  (/* width: 80vw; */)  has been commented

![Screenshot (121)](https://user-images.githubusercontent.com/98436409/201529325-c82b0d7b-cf06-4217-934f-853698200afa.png)

Reply:
Thanks, I will take al look at it